server/src/main/java/org/elasticsearch/cluster/service/ClusterService.java

```diff
@@ -88,7 +88,6 @@ public ClusterService(Settings settings, ClusterSettings clusterSettings, Master
     }
 
     private void setSlowTaskLoggingThreshold(TimeValue slowTaskLoggingThreshold) {
-        masterService.setSlowTaskLoggingThreshold(slowTaskLoggingThreshold);
         clusterApplierService.setSlowTaskLoggingThreshold(slowTaskLoggingThreshold);
```

Comment:
can you move the `CLUSTER_SERVICE_SLOW_TASK_LOGGING_THRESHOLD_SETTING` setting to ClusterApplierService?

Reply:
Yup, see 253ac0b.